x-pack/plugins/ml/public/application/components/ml_embedded_map/_ml_embedded_map.scss

```diff
@@ -0,0 +1,8 @@
+.mlEmbeddedMapContent {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex: 1 1 100%;
+  z-index: 1;
+  min-height: 0; // Absolute must for Firefox to scroll contents
+}
```

Comment:
I think it will be beneficial to have this here as the component can be used in several places outside of Data Visualizer and this is the base styling for the inner most div.